### DISCUSSION TITLE
Creation of Databases

### DISCUSSION THREAD

**Discussion by AndiLeni** [pawelsalawa/sqlitestudio#4488] [General]
Hello,

is it possible to create a sqlite database? I could not find an option besides the "connect" button.

Kind regards

**Comment by pawelsalawa** [comment]
Next to the connext/disconnect buttons there is icon with a database and a plus on it:
![image](https://user-images.githubusercontent.com/481846/183898058-2cb80d52-4d5b-4aac-ab74-20e70a70ae8e.png)

There you can add existing or create a new database.

**Reply by AndiLeni** [reply]
Thanks, I missed this option.
